### PR TITLE
feat: upgrade to Mongoose 6.x and drop Node 10 support

### PR DESCRIPTION
**Pull request by fabian4**
Fix: https://github.com/node-casbin/mongoose-adapter/issues/51

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -2,29 +2,29 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.12.11":
-  version "7.12.11"
-  resolved "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.12.11.tgz#f4ad435aa263db935b8f10f2c552d23fb716a63f"
-  integrity sha512-Zt1yodBx1UcyiePMSkWnU4hPqhwq7hGi2nFL1LeA3EUl+q2LQx16MISgJ0+z7dnmgvP9QtIleuETGOiOH1RcIw==
+"@babel/code-frame@^7.0.0", "@babel/code-frame@7.12.11":
+  "integrity" "sha512-Zt1yodBx1UcyiePMSkWnU4hPqhwq7hGi2nFL1LeA3EUl+q2LQx16MISgJ0+z7dnmgvP9QtIleuETGOiOH1RcIw== sha1-9K1DWqJj25NbjxDyxVLSP7cWpj8="
+  "resolved" "https://registry.npmmirror.com/@babel/code-frame/download/@babel/code-frame-7.12.11.tgz"
```
Comment (hsluoyz) [inline yarn.lock]: Plz still use: registry.npmjs.org

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by hsluoyz** [comment]
@fabian4 plz also fix:

image

**Comment by hsluoyz** [comment]
@fabian4 the CI failed:

image

**Comment by hsluoyz** [comment]
@sapcik plz review

**Review by hsluoyz — APPROVED** [review]

**Review by nodece — APPROVED** [review]

**Pull request merged**